common/persistence/nosql/nosqlplugin/cassandra/tasks_test.go

```diff
@@ -66,6 +66,11 @@ func TestSelectTaskList(t *testing.T) {
 					(*tlDB)["ack_level"] = int64(1000)
 					(*tlDB)["kind"] = 2
 					(*tlDB)["last_updated"] = now
+					(*tlDB)["adaptive_partition_config"] = map[string]interface{}{
```

Comment:
Nonblocking/Nit: It might be be worth doing a round-trip test on the to and from mapping functions to ensure that no fields get dropped in the future.